arangod/Cluster/SynchronizeShard.cpp

```diff
@@ -810,6 +844,7 @@ bool SynchronizeShard::first() {
           "/" + resolver.getCollectionName(collection->id());
     }
 
+#if 0
```

Comment:
maybe add a comment on why this is deactivated

Reply:
Done